src/components/DataSubmissions/DataSubmissionSummary.tsx

```diff
@@ -172,17 +217,26 @@ const DataSubmissionSummary: FC<Props> = ({ dataSubmission }) => {
       >
         <Stack direction="column" minWidth="192px">
           <StyledSubmissionTitle variant="h6">STATUS</StyledSubmissionTitle>
-          <StyledSubmissionStatus variant="h5">
+          <StyledSubmissionStatus variant="h5" aria-label="Data Submission status">
             {dataSubmission?.status}
           </StyledSubmissionStatus>
-          <StyledHistoryButton
-            variant="contained"
-            color="info"
-            fullWidth
-            onClick={handleOnHistoryDialogOpen}
-          >
-            Full History
-          </StyledHistoryButton>
+          <Stack direction="column" spacing="9px" justifyContent="center" alignItems="center">
+            <StyledReviewCommentsButton
+              variant="contained"
+              color="info"
+              onClick={handleOnReviewCommentsDialogOpen}
+              disabled={!lastReview}
+            >
+              Review Comments
+            </StyledReviewCommentsButton>
+            <StyledHistoryButton
+              variant="contained"
+              color="info"
+              onClick={handleOnHistoryDialogOpen}
+            >
+              Full History
+            </StyledHistoryButton>
+          </Stack>
         </Stack>
```

Comment:
Could you confirm if our alignment is correct? Figma seems to have it slightly more centered under the status, whereas ours is offset to the right. Maybe this is just due to the length of the status text?

Ours:

<img width="197" alt="Screenshot 2024-02-16 at 10 03 03 AM" src="https://github.com/CBIIT/crdc-datahub-ui/assets/38357871/8da4fdb1-1b3e-44a8-baca-6f7295237994">

Figma:

<img width="197" alt="Screenshot 2024-02-16 at 10 03 13 AM" src="https://github.com/CBIIT/crdc-datahub-ui/assets/38357871/a2536f3b-04a0-4643-82b6-b13a5a47d124">

Reply:
Thanks for catching these for me, fixed them.